src/js/program/program-controller.js

```diff
@@ -448,12 +448,12 @@ class ProgramController extends Events {
      */
     _setActiveMedia(mediaController) {
         const { model } = this;
-        const { mediaModel, provider } = mediaController;
+        const { mediaModel, provider, mediaElement } = mediaController;
```

Comment:
`mediaElement` is a `get` on `mediaController`, So we don't want to assign it to a variable re: the change above. By calling `mediaController.mediaElement`, we're accessing the `get` function so we'll always have the latest instance of `mediaElement`. Otherwise we're stuck with a static non-updating value that can become stale during various playback cases.